tests/transceiver/infra/paths.py

```diff
@@ -0,0 +1,20 @@
+"""Centralized relative path constants for transceiver attribute infrastructure.
+
+This module consolidates commonly used relative paths so that they can be
+maintained in one place and imported where needed.
+
+Add new constants here rather than redefining them across files.
+"""
+import os
+
+
+# Root directory (relative to repository root)
+REL_TRANSCEIVER_INV_DIR = os.path.join('ansible', 'files', 'transceiver', 'inventory')
```

Comment:
Will it cause issue if the test is run from different directory? Have we thought about using pathlib.Path(__file__) and then traversing from there to get to the root directory?